src/command_modules/azure-cli-vm/tests/test_vm_commands.py

```diff
@@ -593,8 +593,8 @@ def body(self):
             self.resource_group, self.vm_name))
         self._check_vm_power_state('PowerState/deallocated')
         self.cmd('vm resize -g {} -n {} --size {}'.format(
-            self.resource_group, self.vm_name, 'Standard_A4'),
-            checks=JMESPathCheck('hardwareProfile.vmSize', 'Standard_A4'))
+            self.resource_group, self.vm_name, ' Standard_DS2_v2'),
+            checks=JMESPathCheck('hardwareProfile.vmSize', 'Standard_DS2_v2'))
```

Comment:
A* is HDD based, but DS* requires SSD which needs to be backed up by premium_* storage 
Likely, the service side changed that you can't downgrade the premium based vm to standard